iio/iio_app/iio_app.c

```diff
@@ -71,6 +77,9 @@ int32_t iio_app_run(struct iio_app_device *devices, int32_t len)
 	struct wifi_init_param		wifi_param;
 	struct wifi_desc		*wifi;
 	struct uart_desc		*uart_desc;
+#else
```

Comment:
Probably should be `#elif defined (LINUX_PLATFORM)` because like this, if neither `LINUX_PLATFORM` or `USE_TCP_SOCKET` are defined, I think `struct tcp_socket_init_param` doesn't exist. CI signals this error.